README.md

```diff
@@ -8,20 +8,53 @@ Command line tool for PostgreSQL server activity monitoring.
 
 ![pg_activity screenshot](https://raw.github.com/dalibo/pg_activity/master/docs/imgs/screenshot.png)
 
-Installation
-------------
+Installation from packages (recommended)
+----------------------------------------
 
 pg\_activity is available in many Linux distributions; the PostgreSQL Global
 Development Group (PGDG) also provides packages for RPM-based
 (https://yum.postgresql.org/) and Debian-based distributions
-(https://wiki.postgresql.org/wiki/Apt). Using distribution packages is the
-recommended way to install pg\_activity.
+(https://wiki.postgresql.org/wiki/Apt):
+
+    $ sudo yum install pg_activity
+
+    $ sudo apt install pg-activity
+
+Using distribution packages is the recommended way to install pg\_activity.
+
+Installation from pip
+---------------------
 
 Alternatively, pg\_activity can be installed using pip on Python 3.6 or later
 along with psycopg2 (version 2.8 or higher):
 
     $ python3 -m pip install pg_activity psycopg2-binary
 
+Then launch:
+
+    $ ~/.local/bin/pg_activity
```

Comment:
Not sure the info is needed since calling pg_activity works just fine as is.

Reply:
Thanks for  the PR !

Reply:
When installing through pip  with the postgres user on Ubuntu, I have this:

```
  WARNING: The script pg_activity is installed in '/var/lib/postgresql/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
```

I suppose that `.local/bin` is not in the `$PATH` of system users.